packages/eslint-config-typescript-react/src/index.ts

```diff
@@ -4,25 +4,23 @@ require('@rushstack/eslint-patch/modern-module-resolution');
 
 const config: Linter.Config = {
   extends: [
-    require.resolve('eslint-config-airbnb/rules/react'),
-    require.resolve('eslint-config-airbnb/rules/react-a11y'),
-    require.resolve('eslint-config-airbnb/hooks'),
     require.resolve('@rightcapital/eslint-config-typescript'),
+    require.resolve('eslint-config-airbnb/hooks'),
+    require.resolve('eslint-config-airbnb/rules/react-a11y'),
     'plugin:@rightcapital/recommended-react',
+    'plugin:@eslint-react/recommended-legacy',
   ],
   plugins: ['@rightcapital'],
   reportUnusedDisableDirectives: true,
   rules: {
     'jsx-a11y/anchor-is-valid': 'off',
     'jsx-a11y/click-events-have-key-events': 'off',
-    'react/function-component-definition': 'off',
-    'react/jsx-filename-extension': ['warn', { extensions: ['.jsx', '.tsx'] }],
-    'react/jsx-key': ['error'],
-    'react/jsx-no-useless-fragment': ['error', { allowExpressions: true }],
-    'react/jsx-uses-react': 'off',
-    'react/react-in-jsx-scope': 'off',
-    // We are using TypeScript, so it's not necessary to use React's defaultProps
-    'react/require-default-props': 'off',
+    '@eslint-react/naming-convention/filename-extension': [
+      'warn',
+      { extensions: ['.jsx', '.tsx'] },
```

Comment:
`@eslint-react/naming-convention/filename-extension` already defaults to `"extensions": [".jsx", ".tsx"]`. May we omit it?

In contrast, `react/jsx-filename-extension` defaults to `"extensions": [".jsx"]`, thus we are manually set `extensions` in the past.

References:
1. `@eslint-react/naming-convention/filename-extension`: https://eslint-react.xyz/rules/naming-convention-filename-extension
2. `react/jsx-filename-extension`: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-filename-extension.md
3. `react/jsx-filename-extension` issue about `.tsx` extension: https://github.com/jsx-eslint/eslint-plugin-react/issues/2919

Reply:
Okay, I will remove them. Previously, what I considered was to explicitly declare the rule configuration to avoid being affected by upstream changes. 
However, we have corresponding [_config-snapshot.test.ts_](https://github.com/RightCapitalHQ/frontend-style-guide/blob/main/specs/eslint/config-snapshot.test.ts), so even if the default configuration upstream changes, we can still observe it [here](https://github.com/RightCapitalHQ/frontend-style-guide/blob/main/specs/eslint/__snapshots__/config-snapshot.test.ts.snap).